### DISCUSSION TITLE
<res-ref-name> missing java:/ for mail service (WildFly 14)

### DISCUSSION THREAD

**Discussion by mazimmer** [imixs/imixs-jsf-example#60] [General]
Current `jboss-web.xml `doesn't agree with WildFly 14.

Changing to `<res-ref-name>java:/mail/org.imixs.workflow.mail</res-ref-name>` allows WAR to be deployed.

**Comment by rsoika** [comment]
Ok - thanks a lot for that finding. Yes we need to go forward to Wildfly 14.....
I will fix this soon. Can you work with the example in Wildfly 14 ?

**Comment by mazimmer** [comment]
There’s another issue with all the tables missing - seems like eclipselink didn’t issue the DDL commands to create them. Anything I need to configure there, that’s not in the deployment description? The DB user has “all privileges” on the database….

But it works until the code tries to hit a table, i.e. when I try to create a work item.

**Comment by rsoika** [comment]
Take a look here: https://imixs.org/doc/deployment/wildfly.html
You need to provide the eclipselink module for wildfly manually. 

If you use the docker image 'imixs/wildfly" this works out of the box. But we have not yet created a image for wildfly 14. We are still on wildfly 13. But I think the eclipselink configuration has not changed here.

**Comment by mazimmer** [comment]
That’s what I have done - I’ve configured everything according to the deployment guide, but the tables aren’t getting created… But I’ll re-deploy the thing.

Regards, 

Marco

**Comment by rsoika** [comment]
You may also test the JDBC connection from the Wildfly Web Console. (http://localhost:9990). 
If EclipseLink is not generating the tables, a connection failure may be the reason. You can also post your log output if you like.
